### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by maximiliangustmann**
Please review my code!

### CODE REVIEW

File: src/components/Reviews.stories.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export default {
+  title: 'Reviews',
+  component: Reviews,
+}
+
+const Template = (args) => <Reviews {...args} />
+
+export const singleReview = Template.bind({})
+
+singleReview.args = {}
```
Comment (mpagels) [inline src/components/Reviews.stories.js:13]: If you don't want pass args you can leave that out
```suggestion
const Template = () => <Reviews />

export const singleReview = Template.bind({})
```

File: src/components/Reviews.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      {reviews.map((review, index) => (
+        <Review key={index}>
+          <h2>{review.title}</h2>
+          <p>Rating: {review.rating}</p>
+          <h3>{review.categorie}</h3>
+          <h3>{review.subcategorie}</h3>
+          <h3>Summary: </h3>
+          <p>{review.summary}</p>
+          <h3>Lessons: </h3>
+          <p>{review.lessons}</p>
```
Comment (mpagels) [inline src/components/Reviews.js:17]: Not shure if it be better to use h4 for review.subcategorie, or p is missing after h3 review.categorie
looks odd

**Review by mpagels — COMMENTED** [review]
Good job! have found two little things. pls review

File: src/components/Reviews.js (hunk 2 above)
Comment (maximiliangustmann) [reply src/components/Reviews.js]: You are right, changed it!

**Pull request merged**